packages/twenty-server/src/engine/core-modules/message-queue/message-queue.module-factory.ts

```diff
@@ -32,20 +33,12 @@ export const messageQueueModuleFactory = async (
       };
     }
     case MessageQueueDriverType.BullMQ: {
-      const host = environmentService.get('REDIS_HOST');
-      const port = environmentService.get('REDIS_PORT');
-      const username = environmentService.get('REDIS_USERNAME');
-      const password = environmentService.get('REDIS_PASSWORD');
+      const connectionString = environmentService.get('REDIS_URL');
 
       return {
         type: MessageQueueDriverType.BullMQ,
         options: {
-          connection: {
-            host,
-            port,
-            username,
-            password,
-          },
+          connection: new IORedis(connectionString)
```

Comment:
this seems wrong actually, we should not instanciate here

Reply:
true, i changed it to just pass the connection string here. It works but there is a type error. Unsure why that is since passing a string should be OK as per: https://redis.github.io/ioredis/classes/Redis.html#constructor